<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProxySettings</class>
 <widget class="QWidget" name="ProxySettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>204</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QRadioButton" name="proxySystem">
     <property name="text">
      <string>Use the system proxy settings</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="proxyDirect">
     <property name="text">
      <string>Direct internet connection</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="proxyManual">
     <property name="text">
      <string>Manual proxy configuration</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="proxy_manual_container" native="true">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <property name="leftMargin">
       <number>24</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QComboBox" name="proxyType">
          <item>
           <property name="text">
            <string>HTTP Proxy</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>SOCKS Proxy</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="LineEdit" name="proxyHost"/>
        </item>
        <item>
         <widget class="QLabel" name="label_15">
          <property name="text">
           <string>Port</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="SpinBox" name="proxyPort">
          <property name="maximum">
           <number>65535</number>
          </property>
          <property name="value">
           <number>8080</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGroupBox" name="proxyAuth">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="title">
         <string>Use authentication</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
        <layout class="QFormLayout" name="formLayout">
         <item row="0" column="0">
          <widget class="BuddyLabel" name="label_16">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Username:</string>
           </property>
           <property name="buddy">
            <cstring>proxyUsername</cstring>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="LineEdit" name="proxyUsername">
           <property name="enabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="BuddyLabel" name="label_17">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Password:</string>
           </property>
           <property name="buddy">
            <cstring>proxyPassword</cstring>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="LineEdit" name="proxyPassword">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="echoMode">
            <enum>QLineEdit::Password</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>2</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LineEdit</class>
   <extends>QLineEdit</extends>
   <header>lineedit.h</header>
  </customwidget>
  <customwidget>
   <class>SpinBox</class>
   <extends>QSpinBox</extends>
   <header>spinbox.h</header>
  </customwidget>
  <customwidget>
   <class>BuddyLabel</class>
   <extends>QLabel</extends>
   <header>buddylabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>proxyManual</sender>
   <signal>toggled(bool)</signal>
   <receiver>proxy_manual_container</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>35</x>
     <y>61</y>
    </hint>
    <hint type="destinationlabel">
     <x>10</x>
     <y>91</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>proxyAuth</sender>
   <signal>toggled(bool)</signal>
   <receiver>proxyUsername</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>129</x>
     <y>120</y>
    </hint>
    <hint type="destinationlabel">
     <x>125</x>
     <y>148</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>proxyAuth</sender>
   <signal>toggled(bool)</signal>
   <receiver>proxyPassword</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>142</x>
     <y>125</y>
    </hint>
    <hint type="destinationlabel">
     <x>176</x>
     <y>179</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>proxyAuth</sender>
   <signal>toggled(bool)</signal>
   <receiver>label_16</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>204</x>
     <y>123</y>
    </hint>
    <hint type="destinationlabel">
     <x>63</x>
     <y>144</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>proxyAuth</sender>
   <signal>toggled(bool)</signal>
   <receiver>label_17</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>230</x>
     <y>123</y>
    </hint>
    <hint type="destinationlabel">
     <x>83</x>
     <y>181</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>proxyManual</sender>
   <signal>toggled(bool)</signal>
   <receiver>proxyAuth</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>50</x>
     <y>60</y>
    </hint>
    <hint type="destinationlabel">
     <x>102</x>
     <y>118</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
